virtualpdu/tests/integration/pdu/test_pysnmp_handler.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2016 Internap
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import mock
+from pyasn1.type import univ
+from pysnmp.entity.rfc3413.oneliner import cmdgen
+from virtualpdu.pdu import pysnmp_handler
+from virtualpdu.tests import base
+from virtualpdu.tests import snmp_client
+
+
+# TODO(mmitchell): Merge me with test_pdu.py, or the other way around.
```

Comment:
Check out test_pdu, it's almost the same (except it's a real PDU not a mock, but the intent was the same).